src/resources/i18n/ors-dictionary.i18n.cs-cz.js

```diff
@@ -0,0 +1,71 @@
+
+export default {
+  orsDictionary: {
+    unknown: 'Neznámý',
+    paved: 'Zpevněný',
+    unpaved: 'Nezpevněný',
+    asphalt: 'Asfalt',
+    concrete: 'Beton',
+    cobblestone: 'Dlažba',
+    metal: 'Kovový',
+    wood: 'Dřevěný',
+    compactedGravel: 'Zhutněný štěrk',
+    fineGravel: 'Jemný štěrk',
+    gravel: 'Štěrk',
+    dirt: 'Prach',
+    ground: 'Hlína',
+    ice: 'Led',
+    pavingStones: 'Dlažební kostky',
+    sand: 'Písek',
+    woodchips: 'Dřevní štěpka',
+    grass: 'Tráva',
+    grassPaver: 'Travnatý povrch',
+    none: 'Žádný',
+    no: 'Ne',
+    customers: 'Zákazníci',
+    destination: 'Cíl',
+    delivery: 'Obslužná',
```

Comment:
Thanks for the clarification.
When you state "can be used specific word for road that used mostly for commercial delivery services" - does that mean there's another, different word for roads used mostly for commercial delivery services? Because if so, that'd be it.
If I misunderstood and it is not the case, then that works for me.